src/main/java/io/vavr/control/Validation.java

```diff
@@ -111,6 +111,15 @@ private Validation() {
     /**
      * Creates a {@code Validation} of an {@code Either}.
      *
+     * <pre>{@code
+     * // validates the either type if either is right then validation
+     * //contains right value otherwise throws NoSuchElementException
+     * // Creates Validation instance with the value "vavr"
+     *    Validation validation = Validation.fromEither(Either.right("vavr"))
+     * //throws NoSuchElementException
+     *    Validation validation = Validation.fromEither(Either.left("vavr"));
```

Comment:
**I have this feeling that you - USE TABS INSTEAD OF SPACES - right? 🙀😊**

Please
* always use spaces, it is the convention for this repo
* do not add extra indentation at the beginning of a line
* and do one space after `//`

In this particular example:

```java
     * // validates the either type if either is right then validation
     * // contains right value otherwise throws NoSuchElementException
     * // Creates Validation instance with the value "vavr"
     * Validation validation = Validation.fromEither(Either.right("vavr"))
     * // throws NoSuchElementException
     * Validation validation = Validation.fromEither(Either.left("vavr"));
```